Models/OrderPart.cs

```diff
@@ -405,7 +405,7 @@ public string CurrencyCode
             get { return Id + " - " +
                 Status + " - " +
                 BillingAddress.Honorific + " " + BillingAddress.FirstName + " " + BillingAddress.LastName + " - " +
-                Currency.GetPriceAsString(Total, CurrencyCode) +
+                CurrencyCode + " " + Total.ToString() +

```

Comment:
off the top of my head, the best I came up with is `Currency.Currencies[CurrencyCode].PriceAsString(Total, CultureInfo.CurrentCulture)`